backend/battles/models.py

```diff
@@ -81,6 +81,13 @@ class Battle(models.Model):
     def winner(self):
         return get_battle_result(self)
 
+    def has_been_resolved(self):
```

Comment:
I feel like for this to be 100% future proof it should also check for the `creator`'s pockemons

Reply:
Great idea!

Reply:
Refactoring suggestion:
```
return all([
     self.creator_pokemon_1, 
     self.creator_pokemon_2, 
     self.creator_pokemon_3, 
     self.opponent_pokemon_1, 
     self.opponent_pokemon_2, 
     self.opponent_pokemon_13
])
```